lumos/templates/home2.html

```diff
@@ -1,5 +1,4 @@
 <!DOCTYPE html>
-{% load staticfiles %}
 <html>
```

Comment:
Please read up on this

https://docs.djangoproject.com/en/1.9/howto/static-files/